src/Framework/StringBuilderCache.cs

```diff
@@ -51,27 +55,44 @@ public static StringBuilder Acquire(int capacity = 16 /*StringBuilder.DefaultCap
             if (capacity <= MAX_BUILDER_SIZE)
             {
                 StringBuilder sb = StringBuilderCache.t_cachedInstance;
+                StringBuilderCache.t_cachedInstance = null;
```

Comment:
Why move this line? Doesn't this mean you're essentially deallocating t_cachedInstance if it's not null but smaller than capacity?

Reply:
It was moved so line 88 works. There is no point to keep it in thread static variable because this SB is doomed anyway as it will be replaced by Release before anything would have chance to use it.